src/content/docs/fr/guides/integrations-guide/mdx.mdx

```diff
@@ -84,13 +84,127 @@ Pour les autres éditeurs, utilisez le [serveur de langage MDX](https://github.c
 
-Regardez comment MDX fonctionne dans Astro avec des exemples dans notre [Guide Markdown & MDX](/fr/guides/markdown-content/).
+L'intégration de MDX permet d'améliorer la création de documents Markdown avec des variables, des expressions et des composants JSX.
 
-Visitez la [documentation MDX](https://mdxjs.com/docs/what-is-mdx/) pour apprendre à utiliser les fonctions MDX standard.
+Elle ajoute également des fonctionnalités supplémentaires au MDX standard, notamment la prise en charge de la matière première de style Markdown dans le MDX. Cela vous permet d'utiliser la plupart des [fonctionnalités Markdown intégrées à Astro](/fr/guides/markdown-content/).
+
+Les fichiers `.mdx` doivent être écrits en [syntaxe MDX](https://mdxjs.com/docs/what-is-mdx/#mdx-syntax) plutôt qu'en syntaxe HTML d'Astro.
+
+### Utilisation de variables exportées dans MDX
+
+MDX permet d'utiliser les instructions `export` pour ajouter des variables à votre contenu MDX ou pour exporter des données vers un composant qui les importe.
+
+Par exemple, vous pouvez exporter un champ `title` d'une page ou d'un composant MDX pour l'utiliser comme titre avec des `{expressions JSX}` :
+
+```mdx title="/src/pages/posts/post-1.mdx"
+export const title = 'Mon tout premier post MDX';
+
+# {title}
+```
+
+Ou vous pouvez utiliser ce `title` exporté dans votre page en utilisant les instructions `import` et `import.meta.glob()` :
+
+```astro title="src/pages/index.astro"
+---
+const matches = await import.meta.glob('./posts/*.mdx', { eager: true });
+const posts = Object.values(posts);
+---
+
+{posts.map(post => <p>{post.title}</p>)}
+```
+
+#### Propriétés exportées
+
+Les propriétés suivantes sont disponibles pour un composant `.astro` lorsqu'il utilise une instruction `import` ou `import.meta.glob()` :
+
+- **`file`** - Le chemin absolu du fichier (par exemple `/home/user/projects/.../file.mdx`).
+- **`url`** - L'URL de la page (par exemple `/fr/guides/markdown-content`).
+- **`frontmatter`** - Contient toutes les données spécifiées dans le frontmatter YAML du fichier.
+- **`getHeadings()`** - Une fonction asynchrone qui renvoie un tableau de tous les titres (`<h1>` à `<h6>`) dans le fichier avec le type : `{ depth : number ; slug : string ; text : string }[]`. Le `slug` de chaque rubrique correspond à l'ID généré pour une rubrique donnée et peut être utilisé pour les liens d'ancrage.
+- **`<Content />`** - Un composant qui renvoie le contenu complet et affiché du fichier.
+- **(toute valeur `export`)** - Les fichiers MDX peuvent également exporter des données à l'aide d'une instruction `export`.
+
+### Utilisation de variables Frontmatter dans MDX
+
+L'intégration d'Astro MDX inclut la prise en charge de l'utilisation de frontmatter dans MDX par défaut. Ajoutez des propriétés frontmatter comme vous le feriez dans des fichiers Markdown, et ces variables sont disponibles pour être utilisées dans le modèle, et en tant que propriétés nommées lors de l'importation du fichier ailleurs.
```

Comment:
This sentence seems a bit hard to read (I'm tired, it doesn't help), but I'm having a bit of trouble finding a better formulation. Maybe:
```suggestion
L'intégration MDX d'Astro inclut la prise en charge de l'utilisation de frontmatter dans MDX par défaut. Ajoutez des propriétés frontmatter comme vous le feriez dans des fichiers Markdown, et ces variables pourront être utilisées aussi bien dans le modèle qu'en tant que propriétés nommées lorsque vous importez le fichier ailleurs.
```
What do you think?

Reply:
better!